packages/lexical-mark/src/MarkNode.ts

```diff
@@ -168,6 +170,6 @@ export function $createMarkNode(ids: Array<string>): MarkNode {
   return new MarkNode(ids);
 }
 
-export function $isMarkNode(node: ?LexicalNode): boolean %checks {
+export function $isMarkNode(node?: LexicalNode | null): node is MarkNode {
```

Comment:
We want to make sure we pass something to this method: 
```suggestion
export function $isMarkNode(node: LexicalNode | null): node is MarkNode {
```